lib/listing.js

```diff
@@ -27,7 +27,7 @@ const getFileMeta = async filePath => {
 // Exports ---------------------------------------------------------------------
 
 export const renderListFile = async filePaths => {
-  const { title, listingTemplate, theme, assetsDir } = getOptions();
+  const { title, description, listingTemplate, theme, assetsDir } = getOptions();
```

Comment:
Where's `description` coming from? Not sure what it has to do with intention of PR?